doc/releases/changelog-0.35.0.md

```diff
@@ -6,20 +6,150 @@
 
 <h4>Easy to import circuits 💾</h4>
 
-* An error message provides clearer instructions for installing PennyLane-Qiskit if the `qml.from_qiskit`
-  function fails because the Qiskit converter is missing.
+* This version of PennyLane makes it easier to import workflows from Qiskit.
+
+  The `qml.from_qiskit` function converts a Qiskit
+  [QuantumCircuit](https://docs.quantum.ibm.com/api/qiskit/qiskit.circuit.QuantumCircuit) into
+  a PennyLane
+  [quantum function](https://docs.pennylane.ai/en/stable/introduction/circuits.html#quantum-functions).
+  Although `qml.from_qiskit` already exists in PennyLane, we have made a number of improvements to
+  make importing from Qiskit easier:
   [(#5218)](https://github.com/PennyLaneAI/pennylane/pull/5218)
+  [(#5168)](https://github.com/PennyLaneAI/pennylane/pull/5168)
+
+  * You can now append PennyLane measurements onto the quantum function returned by
+    `qml.from_qiskit`. Consider this simple Qiskit circuit:
+
+    ```python
+    import pennylane as qml
+    from qiskit import QuantumCircuit
+
+    qc = QuantumCircuit(2)
+    qc.rx(0.785, 0)
+    qc.ry(1.57, 1)
+    ```
+    
+    We can convert into a PennyLane QNode in just a few lines.
+
+    ```pycon
+    >>> dev = qml.device("default.qubit")
+    >>> measurements = [qml.expval(qml.Z(0) @ qml.Z(1))]
+    >>> qfunc = qml.from_qiskit(qc, measurements=measurements)
+    >>> qnode = qml.QNode(qfunc, dev)
+    >>> qnode()
+    [tensor(0.00056331, requires_grad=True)]
+    ```
```

Comment:
Happy to improve the code, but I'd like to avoid having to do:
```
@qml.qnode(dev)
def f():
    return qml.from_qiskit(qc, measurements)
```
I'd like to emphasize to users that you can avoid this more lengthy explicit-functional style (which may also be unfamiliar to new users).

Reply:
Interesting... ok I'll think on it. It's a subtlety, but worth the thought!